stdlib/public/Darwin/Foundation/String.swift

```diff
@@ -22,10 +22,61 @@
 
 extension String {
   public init(_ cocoaString: NSString) {
-    self = String(_cocoaString: cocoaString)
+    self = String._unconditionallyBridgeFromObjectiveC(cocoaString)
   }
 }
 
+@_effects(releasenone)
+private func _getClass(_ obj: NSString) -> AnyClass {
+  return object_getClass(obj)!
```

Comment:
Ideally we'd use _swift_classOfObjCHeapObject here instead, but I haven't figured out a way to do that yet